packages/types/src/user/common.ts

```diff
@@ -1,5 +1,38 @@
+import { DateComparisonOperator } from "../common"
+
 export type UserDTO = {
   id: string
+  email: string
+  first_name?: string | null
+  last_name?: string | null
+  avatar_url?: string | null
+  metadata?: Record<string, unknown> | null
+  created_at: Date
+  updated_at: Date
+  deleted_at?: Date | null
+}
+
+export type FilterableUserProps = {
+  id?: string | string[]
+  email?: string | string[]
+  first_name?: string | string[]
+  last_name?: string | string[]
 }
 
-export type FilterableUserProps = {}
+export type InviteDTO = {
+  id: string
+  email: string
+  accepted: boolean
+  token: string
+  expires_at: Date
+  metadata?: Record<string, unknown> | null
+  created_at: Date
+  updated_at: Date
+  deleted_at?: Date | null
```

Comment:
todo: The DTO is meant to represent the data that can transfert, deleted at and metadata should be present and non optional. In a later future, we can adapt the response type of our method by dynamically pick from the DTO based on the select and relations

Reply:
same for the UserDTO